merging my branch with main

MacarronGlasses opened the pull request:
As main is pretty empty, and my branch is the most updated one, I think merging then wouldnt be a prob.

TRASHMASTERexe committed:
set up 1st person controller and basic movement

TRASHMASTERexe committed:
fix diagonal movement

TRASHMASTERexe committed:
fix sliding on stop

TRASHMASTERexe committed:
fix in air move speed

MacarronGlasses committed:
update

MacarronGlasses committed:
Changed values in jump formula

TRASHMASTERexe committed:
rewrite move script, remove air straif

TRASHMASTERexe committed:
add air straifing but remove backwards movement while in air

MacarronGlasses committed:
pulled the changes in trashmaster branch

TRASHMASTERexe committed:
init terrain setup

TRASHMASTERexe committed:
fix air straifing add placeholder logic

TRASHMASTERexe committed:
add move speeds for backwards and straif

MacarronGlasses committed:
enemyMov prototype

some files font matter, the obly one that is important is "EnemyMove"

MacarronGlasses committed:
Merge branch 'terrain' into Ktaz

MacarronGlasses committed:
Finished enemy AI and tweaked the map assets a bit

uploading the enemy AI base design working and I delted the cubes GameObjects under the map + used one of them as example to test enemy AI

TRASHMASTERexe committed:
revert to built in render pipeline

MacarronGlasses committed:
Merge branch 'terrain' into Ktaz

MacarronGlasses committed:
changed light intesity in the scene

TRASHMASTERexe committed:
get terrain reworked

MacarronGlasses committed:
Merge branch 'terrain' into Ktaz

MacarronGlasses committed:
Merged

TRASHMASTERexe committed:
finish tunnel entrance/exit

TRASHMASTERexe committed:
add path to scene

TRASHMASTERexe committed:
remove unused assets

MacarronGlasses committed:
nothing important

MacarronGlasses committed:
Merge branch 'terrain' into Ktaz

MacarronGlasses committed:
fixed the memory leak in enemy script

kind of fixed the memory leak in the enemymove script, not perfect but working without as many framedrops as before

TRASHMASTERexe committed:
add water

MacarronGlasses committed:
Time Manager adition

I made a Time manager script and a Time event script to test if the Time manager was working.
I attached the time event script to the cube I was using to test the enemy AI so after 30 secs it starts chasing the player

MacarronGlasses committed:
added a coment in the Time Manager script

MacarronGlasses committed:
Merge branch 'terrain' into Ktaz

MacarronGlasses committed:
Changed scene from terrain

MacarronGlasses committed:
Added another directional light

added a directional light in the 2 entrance of the tunnel, one was mising

TRASHMASTERexe commented:
I dont think we want to merge this if it contains all the terrain code, I do think we probably want to merge the 1st person branch, The terrain code is close to finished at which point we could merge that, which would then make this branch eligible in theory. 

FFR: you dont want to merge branches that are still being worked on, that means if you pull code from working brnaches into your branch you need to wait for those to finish

MacarronGlasses committed:
Implemented Saving System (prototype)

Saving system, it only saves the players and enemy coordinates. It works along side the Time manager.

MacarronGlasses committed:
Changed some variable names and made demonstration

juat tweaking with some variable names for more clean code and made a demosntration of the saving and loading system using the player, after set amount of time the player teleports to the location he was about 10 secs ago

MacarronGlasses committed:
just playing around with the scene

testing some scripts
Note to Self: For some fucking reason when you open Unity the save and load Demosntration kinda breaks and theres no apperant reason why because last time I closed it the teleport was working jut fine, but when I opened unity again, it was kinda buggy. Look into it
